about.md

```diff
@@ -0,0 +1,44 @@
+---
+breadcrumb: About Us
+---
+
+# About Us
+
+Welcome to the 'About Us' page. We are a dedicated participant in the global cybersecurity landscape, contributing our efforts towards the development of reliable encryption tools. Our dedicated team is at the helm of the OpenSSL project, ensuring secure digital interactions. Here, you'll meet the individuals who are collectively shaping a safer, privacy-respecting digital future.
```

Comment:
This is a subset of people. It seems to be the union of OMC and contractors. Should we include the OTC as well? What about committers? Whichever set of people we decide it will inevitably be incomplete so there should be some mention about what this list represents and some acknowledgement of all the other people that contribute in some form.

Reply:
Ok, I will see how I can improve it.